src/components/Column/Column.tsx

```diff
@@ -0,0 +1,49 @@
+import { CSSProperties, useState } from "react";
+import styled from "styled-components";
+
+import Card from "../Card";
+import Ticket from "../../interfaces/Ticket";
+import { ColumnHeader } from "./ColumnHeader";
+import { ColumnFooter } from "./ColumnFooter";
+import { ColumnBody, DraggableColumnBody } from "./ColumnBody";
+
+export interface ColumnProps {
+  title: string;
+  items?: Ticket[];
+  draggable?: boolean;
+  onEditClick?: () => void;
+  onAddTicket?: () => void;
```

Comment:
It was tempting to remove these properties but I assume they will be removed anyway in the future once we integrate real APIs, so I left them as is